content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -2520,53 +2519,287 @@ Optional attributes can be added to the root element `kanban` to customize the v
 <reference/view_architectures/kanban/header>`, :ref:`progressbar
-<reference/view_architectures/kanban/progressbar>`, and :ref:`templates
-<reference/view_architectures/kanban/templates>`.
+<reference/view_architectures/kanban/progressbar>`.
 
-Placeholders are denoted in all caps.
+.. _reference/view_architectures/kanban/templates:
 
-.. _reference/view_architectures/kanban/field:
+`templates`: define cards structure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`field`: display field values
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The `templates` element is used to define the :ref:`QWeb templates <reference/qweb>` that structure
+the kanban cards.
+
+Cards structure definition can be split into multiple templates for clarity, but at least one root
+template `card` must be defined.
 
-The `field` element declares fields to use in the :ref:`templates
-<reference/view_architectures/kanban/templates>`. If the field is simply displayed, it does not need
-to be pre-declared.
+An additional template can be defined: `menu`. If defined, it is rendered inside a dropdown
+that can be toggled with a vertical ellipsis (:guilabel:`⋮`) on the top right of the card.
+
+The templates are written in :ref:`JavaScript QWeb <reference/qweb/javascript>`
 
 .. code-block:: xml
 
    <kanban>
-       <field name="FIELD_NAME"/>
-       ...
+      <templates>
+         <t t-name="card">
+            <field name="name"/>
+         </t>
+      </templates>
    </kanban>
 
-The `field` element can have the following attributes:
+Fields
+******
 
+Inside those templates, the `field` element allows to render a field. It can have the following
+attributes:
+ 
 .. include:: view_architectures/field_attribute_name.rst
 
-.. admonition:: Possible structure and representation of its rendering
+By default, field nodes are replaced by a `span` containing their formatted value, unless the
+`widget` attribute is specified, in which case their rendering and behavior depends on the
+corresponding widget. The `widget` attribute can have different values including:
 
-   .. list-table::
-      :class: o-showcase-table
+   .. attribute:: handle
+     :noindex:
 
-      * - .. image:: view_architectures/kanban_field.svg
-             :align: center
+     Allows reordering records with a drag and drop, using the corresponding field as order.
 
-      * - .. code-block:: xml
+   .. attribute:: kanban_color_picker
+     :noindex:
 
-             <kanban>
-                 <templates>
-                     <t t-name="kanban-box">
-                         <div>
-                             <field name="name"/>
-                         </div>
-                     </t>
-                 </templates>
-             </kanban>
+     Allows editing a color (integer) field. Combined with the root attribute `highlight_color`,
+     allows editing the color of the cards.
+
+See the :ref:`Field section <reference/js/widgets>` to discover
+various widgets and their options.
+
+Rendering Context
```

Comment:
very nice. we need to explain what the rendering context is in the case of a kanban template.